scripts/install.py

```diff
@@ -69,40 +72,45 @@
 # for use in `download_from_pypi`
 from distutils.version import LooseVersion
 
-from fnmatch import fnmatch
-
 import os
 from os import path
 
 import re
 
 import shutil
 
-from subprocess import call, check_call, check_output, CalledProcessError
+from subprocess import check_call, check_output, CalledProcessError
 
 from urllib2 import urlopen
 
+locale.setlocale(locale.LC_ALL, 'C')
```

Comment:
This will not solve #530: it sets the locale for the current UNIX process, but whenever we spawn another one running `pip` or `conda` or whatever, they will pick the locale as dictated by the environmental variables.

So the right fix would rather be to use `os.setenv('LC_ALL', 'C')` (or similar) --- but then the difficulty in solving issue #530 really is reproducing the problem and making sure that the fix does indeed fix it :-)